packages/react-widgets/src/models/index.js

```diff
@@ -4,4 +4,4 @@ export { getCategories } from './CategoryModel';
 export { geocodeStreetPoint } from './GeocodingModel';
 export { getScatter, HARD_LIMIT as SCATTER_PLOT_HARD_LIMIT } from './ScatterPlotModel';
 export { getTimeSeries } from './TimeSeriesModel';
-export { getTable, paginateTable, HARD_LIMIT as TABLE_HARD_LIMIT } from './TableModel';
```

Comment:
@aaranadev @menusal we've removed these exports because are no longer necessary. Please confirm us that PS are not using it

Reply:
@padawannn Confirmed for our side. you can remove